code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/PushTemplateConstants.kt

```diff
@@ -105,13 +105,13 @@ internal object PushTemplateConstants {
         const val CAROUSEL_LAYOUT = "adb_car_layout"
         const val CAROUSEL_ITEMS = "adb_items"
         const val EXPANDED_BODY_TEXT = "adb_body_ex"
-        const val EXPANDED_BODY_TEXT_COLOR = "adb_clr_body"
+        const val BODY_TEXT_COLOR = "adb_clr_body"
         const val TITLE_TEXT_COLOR = "adb_clr_title"
         const val SMALL_ICON_COLOR = "adb_clr_icon"
-        const val NOTIFICATION_BACKGROUND_COLOR = "adb_clr_bg"
+        const val BACKGROUND_COLOR = "adb_clr_bg"
         const val REMIND_LATER_TEXT = "adb_rem_txt"
-        const val REMIND_LATER_EPOCH_TIMESTAMP = "adb_rem_ts"
```

Comment:
Some of these will also need to be accessible from the caller extension. We will need to decide which ones to move to `PushTemplateIntentConstants`

Reply:
Please see comment instead
https://github.com/adobe/aepsdk-ui-android/pull/8#issuecomment-2125848322

Reply:
Planning to take the changes to the PushTemplateIntentConstants in my next PR